inc/gateways/class-stripe-connect-gateway.php

```
@@ -0,0 +1,1112 @@
+	 */
+	public function process_checkout($payment, $membership, $customer, $cart, $type) {
+		// Use parent's logic but with Connect headers
+		$stripe_client = $this->get_stripe_connect_client();
+
+		// Set the stripe account for Connect
+		if (! empty($this->stripe_connect_account_id)) {
+			$stripe_client->setStripeAccount($this->stripe_connect_account_id);
+		}
+
+		// Add application fee if configured
+		$application_fee_percent = $this->application_fee_percentage;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$application_fee_percent'. (undefined)

(UnusedLocalVariable)